src/ergw_aaa_radius.erl

```diff
@@ -74,6 +80,14 @@ session_auth_options('PAP', Session, Attrs) ->
 session_auth_options(_, _Session, Attrs) ->
     Attrs.
 
+start_authentication(From, Session,
+                     State = #state{disabled_auth = true,
+                                    auth_server = {_, _, Secret}}) ->
+    SessionOpts0 = to_session([{'TLS-Pre-Shared-Key', Secret}]),
```

Comment:
you can't use the RADIUS shared secret as TLS-Pre-Shared-Key. It's probably better to omit it completely.